generated/nifake_non_ivi/nifake_non_ivi_service.cpp

```diff
@@ -75,7 +75,7 @@ namespace nifake_non_ivi_grpc {
       auto handle_grpc_session = request->handle();
       FakeHandle handle = session_repository_->access_session(handle_grpc_session.id(), handle_grpc_session.name());
 
-      auto initiating_session_id = session_repository_->resolve_session_id(handle);
+      auto initiating_session_id = session_repository_->access_session_id(handle_grpc_session.id(), handle_grpc_session.name());
```

Comment:
Why the change?
There can be a one to many relationship between (grpc_session, RFmx handle). Can't look up a session_id by the instruemnt_handle.

Why is it safe?
Similar to the `remove_session` comment, we have the `grpc_session` input from the client so we should just be able to use that to get the actual session_id stored in the session_repository/session_resource_repository.